ecocode-rules-specifications/src/main/rules/EC14/EC14.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
@zippy1978 ... la suppression de tous les fichiers de cet PR est bien voulue ? car doublon avec d'autres rules, c'est ça ?

Reply:
Yes!
The rules deleted where originally added by @gaelfoppolo at the very beginning of the repo. But after that they were re-added with other ids.
So the goal is t clean up the old ones (used by no plugin at all!), and keep the new ones (used by android plugin, and now iOS)